themes/root/theme.config.php

```diff
@@ -1,6 +1,7 @@
 <?php
 return [
     'extends' => false,
+    'css' => ['icons.css'],
```

Comment:
While I see the value in having some kind of global default, I also wonder if it would be better for now to move this into bootstrap3. Adding an additional HTTP transaction for a separate CSS file on every page just for icons seems unnecessary, and out of keeping with other aspects of our CSS management.

Reply:
I agree this is problematic. I also worry about including external dependencies (like FontAwesome's CSS). It would be safe to bundle thru theme.config.php because it will affect nearly pages but if we can make it a one-stop config edit, I'd rather do that.

Reply:
In theory, you could dynamically include CSS through the icon-specific templates, since those inclusions will be deduplicated by the framework, and then we'd only load the styles when they are needed... but that might also be incredibly inefficient on pages with lots of icons, since it would trigger lots of deduplication work. Just a thought for consideration!

Reply:
I've moved the dependency loading to the icon templates but I was also concerned about deduplication work. It turns out, `appendStylesheet` just appends to the end of Container and the deduplication only happens once when it's time to render.

Reply:
That sounds like good news, though it probably wouldn't hurt to do some profiling when this is all ready.